wal

```diff
@@ -298,6 +306,7 @@ export_colors() {
     export_rofi
     export_emacs
     export_plain     > "${cache_dir}/colors"
+    export_putty     > "${cache_dir}/wal.reg"
```

Comment:
Can we name the file `colors.reg` or `colors-putty.reg` to match the other filenames?